engine/src/flutter/lib/web_ui/lib/painting.dart

```diff
@@ -443,6 +443,7 @@ class ColorFilter implements ImageFilter {
   const factory ColorFilter.matrix(List<double> matrix) = engine.EngineColorFilter.matrix;
   const factory ColorFilter.linearToSrgbGamma() = engine.EngineColorFilter.linearToSrgbGamma;
   const factory ColorFilter.srgbToLinearGamma() = engine.EngineColorFilter.srgbToLinearGamma;
+  const factory ColorFilter.saturation(double saturation) = engine.EngineColorFilter.saturation;
```

Comment:
Would be great if Github did not lose said comment. 😅 
See engine/src/flutter/lib/web_ui/lib/src/engine/color_filter.dart
You need to define EngineColorFilter.saturation
Search for EngineColorFilter.srgbToLinearGamma as an example.